Makefile

```diff
@@ -49,3 +56,19 @@ dev:
 	@echo "Running with local development themes"
 	docker run --env-file .env -it --rm -v "$(PWD)/render:${WORKDIR}/render" -v "$(PWD)/templates:${WORKDIR}/templates" ${ORG}/${NAME} ${FLAGS}
 	python -m webbrowser -t "file://$(PWD)/render/index.html"
+
+# Running locally 
+# We optionally include our .env file if it exists (compatibility with the docker
+# --env-file) and then create a vitualenv as needed for running our local command.
+ifneq (,$(wildcard ./.env))
+    include .env
+    export
+endif
+
+$(PYTHON_BIN):
```

Comment:
Mostly for my education: why break this up instead of putting it all in `local` target?

Reply:
Once the PYTHON_BIN is configured, it doesn't need to create the VM or install the pip requirements each time. By breaking it into its own step, it will only need to run once.

Reply:
Makes sense!